Editor/DeploymentScenarios.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace AmazonGameLift.Editor
+{
+    public enum DeploymentScenarios
+    {
+        SingleRegion = 1,
+        SpotFleet = 3,
+        FlexMatch = 4
```

Comment:
These should be 1, 2, and 3 since there are only 3 scenarios going forwards.

Reply:
Changed to 0, 1, 2 to index once deployers are loaded. Can change to 1,2,3 and change logic at that section if required

Reply:
> Changed to 0, 1, 2 to index once deployers are loaded

Why does this need to happen after deployers are loaded? What's the significance of the numbering? This should be set accurately here. 0, 1, 2 or 1, 2, 3 either work.

Reply:
Also it's still 1,3, 4 and not 0,1, 2. Am i missing something